### ISSUE TITLE
Tweak glossary definition for "images of text"

### ISSUE THREAD

**Issue by patrickhlauke** [w3c/wcag#264]
(prompted by https://lists.w3.org/Archives/Public/w3c-wai-gl/2017JanMar/0642.html)

https://www.w3.org/TR/WCAG21/#dfn-images-of-text currently reads

> text that has been rendered in a non-text form (e.g., an image) in order to achieve a particular visual effect

This definition arguably limits the applicability of 1.4.3 to only apply to images of text where the intent of the author was to achieve a particular visual effect - what about images of text where the author had some other reason to use an image of text? Without actually trying to apply any particular effect, maybe the author simply thought it was the best thing to do? Or their tool/development environment only allowed images to be used, rather than actual HTML text?

I'd tweak the definition here to make the second part an example of intent, rather than seeming to imply it's the only intent that matters.

> text that has been rendered in a non-text form (e.g., an image); often, this is done in order to achieve a particular visual effect

**Comment by patrickhlauke** [comment]
further the note

> Note: This does not include text that is part of a picture that contains significant other visual content.

seems like a get-out clause. what if the text is essential to understand the rest of the picture? or if it presents a separate piece of information that the other parts of the picture? why would it then be exempted, if a user's inability to properly perceive it will result in the user not getting the information/content it conveys (in the case of contrast)?

**Comment by DavidMacDonald** [comment]
We should also say something in the understanding about our opinion on text that is in a timed media. Reasons that I haven't been watching videos to fail text contrast under 1.4.3 is because:

1) It would be a significant requirement, but it text in animations and movies is not mentioned in the understanding, which limits itself to "pixels" and static images.
2) The video was not likely rendered in non-text form for the purpose of achieving a particular effect on the text. It was probably made for a different reason.
3) I don't remember any discussion of text contrast in videos during the creation of WCAG 1.4.3.

**Comment by jasonjgw** [comment]
+1 to expanding the scope of what is covered by the definition, as proposed. The purpose of using the image of text has nothing to do with determining whether there is a need for proper contrast or whether it should be provided to the user.

**Comment by alastc** [comment]
I created an update for this, as suggested by Patrick.

It doesn't address the time-based media (which would be better done in the understanding doc), but would apply as an errata...

**Comment by alastc** [comment]
@patrickhlauke there was a good reason for the note, I've updated it in the PR: 
> This does not include **incidental** <a>text</a> that is part of a picture that contains significant other visual content, **such as a picture which includes street signs**.

**Comment by patrickhlauke** [comment]
yup, making sure it says that it's explicitly about *incidental* text makes it clearer, thanks

**Comment by patrickhlauke** [comment]
this leaves the normative definition still as "in order to achieve a particular visual effect". and leaves a loop where the intent arguably wasn't to achieve a particular visual effect, but just because...the author didn't know any better/thought it was the right thing to do. It's kind of trying to imply author intent / making it part of the normative definition, when really author intent here doesn't matter.

so i'd still like to see at least the first change that was proposed in https://github.com/w3c/wcag/pull/777 (which mitigates this to "Often, this is done...")

**Comment by fstrr** [comment]
This was closed by pull request [#777](https://github.com/w3c/wcag/pull/777).

**Comment by patrickhlauke** [comment]
More precisely, #777 was rejected/closed, and some tweaks were made in some understanding document(s)

**Comment by patrickhlauke** [comment]
Also, not sure why the discussion at the time in the call then moved to the understanding for 1.4.3 Contrast, when this was about 1.4.5 Images of Text. ... but whatever.
